packages/truffle/test/scenarios/genesis_time/genesis_time_invalid.js

```diff
@@ -1,50 +0,0 @@
-const MemoryLogger = require("../MemoryLogger");
```

Comment:
Can you explain in more detail just why you deleted these two tests?  I'm not sure I follow.

Reply:
Hm, I was going to ask if @gnidan knows any more about this, but it looks like the PR that originally added this option was #2236, an external contribution reviewed only by Cruz.  So, uh, I guess nobody here anymore is likely to know much about this... :-/

Reply:
@cds-amal and I figured it out that if we are providing a default value for time, then the `genesis_time_undefined.js` doesn't make any sense since it will get a default value of current timestamp whenever time is not configured in the `truffle-config.js`. However, if we are going to pass an undefined value as default for time as suggested by @haltman-at , then `genesis_time_undefined.js` test is going to make sense.

Reply:
And `genesis_time_invalid.js` test looks for a default value and fails the test if there is none. So, if we want to remove `config.genesis_time`(which is always undefined) since we are now passing undefined as the default value when there is no defined value for time in the `truffle-config.js`, then we probably need to remove this test to pass the CI.